server/src/main/resources/org/opensearch/bootstrap/test-framework.policy

```diff
@@ -170,6 +170,7 @@ grant codeBase "file:${gradle.worker.jar}" {
 grant {
   // since the gradle test worker jar is on the test classpath, our tests should be able to read it
   permission java.io.FilePermission "${gradle.worker.jar}", "read";
+  permission java.io.FilePermission "${{gradle.user.home}}/-", "read";
```

Comment:
I do believe this introduce unnecessary complexity, the env variables as supported by security policies do not support indirection. Since Gradle runtime is part of the build itself, we do have options to add trusted classes etc in `BootstrapForTesting` to have such exceptions.